### PR TITLE
Upgrade Youtube connector to v2

### PR DESCRIPTION
**Pull request by janbaykara**
I'm not sure why this hasn't already been done. I'm also not sure why the v1 connector was calling the API, other than to check the category, but that cat info is now in the HTML.

### CODE REVIEW

**Comment by janbaykara** [comment]
Fixes #535

File: connectors/v2/youtube.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+/* global Connector */
+
+var scrobbleMusicOnly = false;
+chrome.storage.local.get('Connectors', function(data) {
```
Comment (inverse) [inline connectors/v2/youtube.js]: Should this be using the storage abstraction?

Comment (janbaykara) [inline connectors/v2/youtube.js]: I believe not, as the connector is run content-side, so does not have access to the background environment objects like ChromeStorage.

Comment (david-sabata) [inline connectors/v2/youtube.js]: @janbaykara you're right. The storage abstraction is not injected into page. I don't think it's necessary and we should probably inject as many scripts as possible (yet still convenient for connectors impl.)

I'm thinking about unified configuration getter for current connector somewhere in base connector, but this is totally ok now ;-)

**Comment by david-sabata** [comment]
The API was there because we couldn't tell if the video is from music category. But I hated it :smile: . Awesome that you got rid of it!

**Pull request merged**